pkg/controller/kibana/controller.go

```diff
@@ -60,8 +62,8 @@ var (
 )
 
 func (r *ReconcileKibana) Reconcile(request reconcile.Request) (reconcile.Result, error) {
-
-	if err := kibana.ReconcileKibanaInstance(request, r.client); err != nil {
+	esClient := elasticsearch.NewClient(constants.ElasticsearchInstanceName, request.Namespace, r.client)
```

Comment:
see comment about ES constant name above.
we should probably do a `get` for `elasticsearch` objects in the request namespace (technically, Kibana does nothing without an ES cluster in the same namespace anyways)